### PR TITLE
uma.pure-db.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [X] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/138194

### Add your comment and screenshots

 None.

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Review by Yuki2718 — COMMENTED** [review]
~~The only rule required on my end is `uma.pure-db.com#@#div[id^="div-gpt-"]`. (The scriptlet is sure obsolete and can be removed.)~~ Actually no additonal rule is required, only removing the obsolete scriptlet rule works. I'll directly fix.

File: JapaneseFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -8,15 +8,19 @@
 ! Bad: ||ad.doubleclick.net^ (should be in adservers.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/138194
+! https://github.com/AdguardTeam/AdguardFilters/issues/137055
+@@||uma.pure-db.com^$generichide
+@@||uma.pure-db.com/ad_$subdocument
+@@||imp-adedge.i-mobile.co.jp/script/*/spot.js$domain=uma.pure-db.com
+@@||i-mobile.co.jp/api/*/spot.ashx$domain=uma.pure-db.com
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=uma.pure-db.com
```
Comment (piquark6046) [inline JapaneseFilter/sections/antiadblock.txt:17]: Yes. I can confirm that these change is unnecessary during reconfirmation.

**Review by piquark6046 — COMMENTED** [review]
But, on my side, `uma.pure-db.com#@#div[id^="div-gpt-"]` is not required.

**Comment by piquark6046** [comment]
> I'll directly fix.

I will close this PR. @Yuki2718

**Pull request closed without merging**